letsdrill.rb

```diff
@@ -1,13 +1,21 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
-end
+	#Put your code here!
+	case integer	
```

Comment:
looks ok. check out @mikegee's PR review. there's a few pointers in there that you can pick up. https://github.com/paircolumbus/LetsDrill/pull/75